rust/cyclotron-core/src/ops/meta.rs

```diff
@@ -6,18 +6,26 @@ use crate::{
     DEAD_LETTER_QUEUE,
 };
 
-pub async fn count_total_waiting_jobs<'c, E>(executor: E) -> Result<u64, QueueError>
+pub async fn count_total_waiting_jobs<'c, E>(executor: E) -> Result<Vec<(u64, String)>, QueueError>
 where
     E: sqlx::Executor<'c, Database = sqlx::Postgres>,
 {
-    let res = sqlx::query!(
-        "SELECT COUNT(*) FROM cyclotron_jobs WHERE state = 'available' AND scheduled <= NOW()",
+    struct Count {
+        count: Option<i64>,
```

Comment:
Seems harmless, but is it really possible for a count/group by to come back with a null?

Reply:
According to sqlx 🤷